product_brand/__openerp__.py

```diff
@@ -32,13 +32,15 @@
     'name': 'Product Brand Manager',
     'version': '0.1',
     'category': 'Product',
-    'summary': ' brand',
-    'author': 'NetAndCo, Akretion, Prisnet Telecommunications SA',
+    'summary': 'Add brand to products',
+    'author': (
+        'NetAndCo, Akretion, Prisnet Telecommunications SA, MONK Software'
+    ),
```

Comment:
can you add 'Odoo Community Association (OCA)' so that the module shows up on apps.odoo.com undere the OCA?